docs/data_collection.rst

```diff
@@ -0,0 +1,99 @@
+Data Collection
+===============
+
+This document describes how snippets data is collected, stored and used.
+
+Snippets follows Mozilla's established best practices for data collection
+and storage of personally identifiable information.
+
+Data Collection: Impressions
+----------------------------
+
+Immpression data is sampled at a rate of ~10%. For a tracked impression the following
+information is sent to the stats server.
+
+.. code-block:: json
+
+    snippet_name
+    locale
+    country
+    metric
+    campaign
+
+
+This information is sent via ajax request to https://snippets-stats.mozilla.org/ . Metric for the above
```

Comment:
I don't understand what is meant by "Metric for the above will always be impression".